pages/dashboard/settings.vue

```diff
@@ -123,5 +185,15 @@ label {
     flex: 3;
     height: fit-content;
   }
+  button {
+    :hover {
+      cursor: pointer;
+    }
+    height: fit-content;
+    flex: 1;
+  }
+}
+.header-top {
+  font-size: 125%;
```

Comment:
Why is this needed?

Reply:
Its actually not used. So i'll delete it.